### DISCUSSION TITLE
How to replace existing proxies in timeline by BRAW file

### DISCUSSION THREAD

**Discussion by chrystelenicot** [latenitefilms/BRAWToolbox#169] [Support]
<img width="2033" alt="Capture d’écran 2024-02-08 à 14 30 42" src="https://github.com/latenitefilms/BRAWToolbox/assets/105152585/8b40493a-96ec-4fc6-960f-c2fba5655c1a">
<img width="1637" alt="Capture d’écran 2024-02-08 à 14 26 52" src="https://github.com/latenitefilms/BRAWToolbox/assets/105152585/32483a66-e483-4f86-88dc-f9f6e6457171">
<img width="1643" alt="Capture d’écran 2024-02-08 à 14 26 45" src="https://github.com/latenitefilms/BRAWToolbox/assets/105152585/025745db-babf-47ec-b130-cbe3312e7fa8">

Hello,

I'm trying to replace my proxies files on my timeline for BRAW:

I dragged the brawtool box effect on my shot (through > OPEN SHOT and not directly on the time line as suggested) and selected the correct path with Select Braw file. I can see it was linked correctly as on meta data there is the good path the Volumes/...until my .braw
However on informations details I can see this is not my file and still the proxy as the size is not correct 1920x1080 instead of 4608x2592
same on my event timeline.

Any idea what I might have did wrong ?

ps: I also did the Relink BRAW Clips within an event etc, but that simply didn't work out.

many thanks in advance.
Chrystele

**Comment by latenitefilms** [comment]
BRAW Toolbox will be reading the native/direct BRAW footage at whatever decode quality you selected, and Final Cut Pro will be scaling it down to fit your existing clip (i.e. 1920x1080). If your final deliverable is 1920x1080 anyway, there shouldn't be any quality difference.

Unfortunately the "Add BRAW Toolbox to Proxy Clips within an EVENT" Toolbox has some issues/limitations, as discussed here:

https://github.com/latenitefilms/BRAWToolbox/issues/124#issuecomment-1532247236

**Comment by chrystelenicot** [comment]
Thanks for your answer. Unfortunately I need to export 4k 4096x2160. Could there is a way to force  this ?
If not I don’t see the point to replace proxies by Braw if I cannot get full resolution.

**Reply by latenitefilms** [reply]
If you have a 4K timeline and export at 4K with full decode quality, then it should do all the BRAW processing in 4K. Is it not?

**Comment by chrystelenicot** [comment]
I'm gonna try it out and keep you informed :)
